content/en/synthetics/api_tests/dns_tests.md

```diff
@@ -37,7 +37,9 @@ DNS tests can run from both [managed](#select-locations) and [private locations]
 
 After choosing to create a `DNS` test, define your test's request.
 
-### Define request
+{{% synthetics-api-tests-templates %}}
+
+### Define request (build from scratch)
```

Comment:
I don't really think you need two verbs in this one heading - it makes it a bit harder to parse than if it were more concise. Can you just say "Build a request from scratch"? I'm not sure how important it is to also have "define" here.